src/lib.rs

```diff
@@ -500,24 +505,19 @@ mod io {
         }
     }
 }
-//
-//
-//
+
 /// An extension trait for serialport::SerialPortBuilder
 ///
-/// This trait adds two methods to SerialPortBuilder:
+/// This trait adds one method to SerialPortBuilder:
 ///
 /// - open_async
-/// - open_native_async
```

Comment:
let's actually use the `open_native_async` method name instead of `open_async`.  It mirrors more correctly the convention of the `serialport-rs` crate and I've already made the same change in `mio-serial`

Reply:
Alright, did that